app/scripts/metamask-controller.test.js

```diff
@@ -175,21 +175,6 @@ describe('MetaMaskController', function () {
     });
   });
 
-  describe('#getAccounts', function () {
-    it('returns first address when dapp calls web3.eth.getAccounts', async function () {
-      const password = 'a-fake-password';
-      await metamaskController.createNewVaultAndRestore(password, TEST_SEED);
-
-      metamaskController.networkController._baseProviderParams.getAccounts(
```

Comment:
This test was broken because it referenced the private variable `_baseProviderParams`, which was removed. I deleted the entire test rather than fix it because I don't understand what value it provided. This is not the place to test network methods.